docker/importer/volume/openhim-import.json

```diff
@@ -1,52 +1,61 @@
 {
-  "Users": [
+  "Channels": [
     {
-      "groups": [
-        "admin"
+      "methods": [
+        "GET",
+        "POST",
+        "DELETE",
+        "PUT"
       ],
-      "firstname": "Super",
-      "surname": "User",
-      "email": "root@openhim.org",
-      "passwordAlgorithm": "sha512",
-      "passwordHash": "ea3824f17cf1379eb118a36bc7c8cf0f45712e2af7748567fca5313dec6fa66d61064e82a5e5cb88e998486ee3c7d0dac235bbeda8c341d6edc1c77406be2ab6",
-      "passwordSalt": "d4f622c0404f09bd959bfb263efa3452",
-      "expiry": null,
-      "locked": false,
-      "token": null,
-      "tokenType": null
-    }
-  ],
-  "Clients": [
-    {
-      "roles": [
+      "type": "http",
+      "allow": [
         "instant"
       ],
-      "customTokenID": "test",
-      "clientID": "test",
-      "name": "Test Client"
-    }
-  ],
-  "Mediators": [
-    {
-      "urn": "covid19-immunization-tracking-mediator",
-      "version": "2.2.2",
-      "name": "Mapping Mediator",
-      "description": "Generic OpenHIM Mapping Mediator",
-      "endpoints": [
+      "whitelist": [],
+      "authType": "private",
+      "matchContentTypes": [],
+      "properties": [],
+      "txViewAcl": [],
+      "txViewFullAcl": [],
+      "txRerunAcl": [],
+      "status": "enabled",
+      "rewriteUrls": false,
+      "addAutoRewriteRules": true,
+      "autoRetryEnabled": false,
+      "autoRetryPeriodMinutes": 60,
+      "routes": [
         {
           "type": "http",
           "status": "enabled",
           "forwardAuthHeader": false,
-          "name": "Generic Mapping Mediator",
+          "name": "Hapi Fhir",
+          "secured": false,
           "host": "localhost",
```

Comment:
@bradsawadye Looks good man. Is there any reason why we are not using the docker compose service name for the FHIR server here? I believe it's `hapi-fhir`, the service name

Reply:
@de-laz thanks for the feedback. The issue has been addressed in https://github.com/jembi/covid19-immunization-tracking-package/pull/2/commits/a4db5873c692af2983bd13b3047fbf1a2b7cede2

Reply:
@de-laz thanks for the feedback. The issue has been addressed in https://github.com/jembi/covid19-immunization-tracking-package/pull/2/commits/a4db5873c692af2983bd13b3047fbf1a2b7cede2